functions-methods.js

```diff
@@ -33,4 +62,20 @@
 // checkEmailValidity("tessmellink@novi.nl") geeft true - want @ en punt op de juiste plek
 // checkEmailValidity("n.eekenanovi.nl") geeft false - want geen @
 // checkEmailValidity("n.eeken@novinl.") geeft false - want de punt mag niet als laatst
-// checkEmailValidity("tessmellink@novi,nl") geeft false - want er staat een komma in
\ No newline at end of file
+// checkEmailValidity("tessmellink@novi,nl") geeft false - want er staat een komma in
+
+function checkEmailValidity(emailAdres) {
+    const containAt = emailAdres.includes('@');
+    const containComma = emailAdres.includes(',');
+    const containDot = emailAdres.lastIndexOf('.');
+    const containDotAtTheEnd = containDot !== emailAdres.length -1;
+
+    if (containAt && !containComma  && containDotAtTheEnd) {
+        return 'true';
+    }else {
+     return 'false';
+    }
```

Comment:
Je kan de uitkomst van de expressie meteen teruggeven:
```suggestion
    return containAt && !containComma  && containDotAtTheEnd
```